config-connector/solutions/networking/helm/autoneg/README.md

```diff
@@ -0,0 +1,173 @@
+    ```
+
+1. Wait for clusters to be created
+
+    ```bash
+    # The command uses cluster names based on the values passed in the ealier step
+    kubectl wait --for=condition=Ready containercluster/cluster-na containercluster/cluster-eu
+    ```
+
+1. Configure first cluster
+
+    ```bash
+    # switch the context to the first cluster. The command uses cluster name and zone based on the values used to create the clusters.
+    gcloud container clusters get-credentials cluster-na --zone=us-central1-b
+
+    # validate your chart
+    helm lint ./workload/ --set projectId=[PROJECT_ID] --set localMessage="Hello from North America\!"
+
+    # install your chart
+    helm install ./workload/ --set projectId=[PROJECT_ID] --set localMessage="Hello from North America\!" --generate-name
+
+     # annotate service account
+    kubectl annotate sa -n autoneg-system default iam.gke.io/gcp-service-account=autoneg-system@[PROJECT_ID].iam.gserviceaccount.com
+
+    # check the service and ensure that `anthos.cft.dev/autoneg-status` annotation is present in the output
+    get svc node-app-backend -o=jsonpath='{.metadata.annotations}'
+
+    # ensure pods are ready
+    kubectl wait --for=condition=Ready pods --all
+
+    # check the service and ensure that `anthos.cft.dev/autoneg-status` annotation is present in the output
+    get svc node-app-backend -o=jsonpath='{.metadata.annotations}'
+    ```
+
+1. Repeat the step for the other cluster
+
+    ```bash
+    # switch the context to the second cluster. The command uses cluster name and zone based on the values used to create the clusters.
+    gcloud container clusters get-credentials cluster-eu --zone=europe-west2-a
+
+    # validate your chart
+    helm lint ./workload/ --set projectId=[PROJECT_ID] --set localMessage="Hello from Europe\!"
+
+
+    # install your chart
+    helm install ./workload/ --set projectId=[PROJECT_ID] --set localMessage="Hello from Europe\!" --generate-name
+
+     # annotate service account
+    kubectl annotate sa -n autoneg-system default iam.gke.io/gcp-service-account=autoneg-system@[PROJECT_ID].iam.gserviceaccount.com
+
+    # ensure pods are ready
+    kubectl wait --for=condition=Ready pods --all
+
+    # check the service and ensure that `anthos.cft.dev/autoneg-status` annotation is present in the output
+    get svc node-app-backend -o=jsonpath='{.metadata.annotations}'
+    ```
+
+1. Switch the context to the cluster that contains the configs for load balancing resources and run verify that multi-cluster ingress is configured
+
+    ```bash
+    # switch the context to the main cluster
+    gcloud container clusters get-credentials [CLUSTER NAME] --zone=[CLUSTER ZONE]
+
+    # if you created the load balancing resources in the namespace, other than default, switch the context to that namespace
+
+    # verify that your backend service has 2 backends attached (select index of "global" if prompted)
+    gcloud compute backend-services describe node-app-backend-service
```

Comment:
Thank you for adding the additional check, Alex. I tried it, and it looks like my backend services look normal, however I still can't curl the IP address. I checked my load balancing resource in GCP Console, it also looks pretty normal. Maybe we should chat offline to do some debugging together?

Reply:
Sure. I am adding additional information showing how backend service output should look like. Also does this step return correct output for you for both clusters?
```
# check the service and ensure that `anthos.cft.dev/autoneg-status` annotation is present in the output
get svc node-app-backend -o=jsonpath='{.metadata.annotations}'
```
Thanks for offering to help debug! I'll set up a some time to look at it together.

Reply:
@maqiuyujoyce , thanks for helping to debug. I've followed up separately on the details of the error I'm seeing with your setup. I've made changes to the instructions above to enable `cloudresourcemanager.googleapis.com`. 
Can you please try re-running the solution on a new project with this API enabled?